src/app/test/helpers/faker.ts

```diff
@@ -0,0 +1,192 @@
+  ids: () => randomArray(0, 5, () => faker.random.number(100)),
+  imageUrl: () => faker.image.imageUrl(),
+  imageUrls: () =>
+    new Array<ImageURL>(
+      {
+        size: "extralarge",
+        url: faker.image.imageUrl(300, 300),
+        width: 300,
+        height: 300,
+      },
+      {
+        size: "large",
+        url: faker.image.imageUrl(220, 220),
+        width: 220,
+        height: 220,
+      },
+      {
+        size: "medium",
+        url: faker.image.imageUrl(140, 140),
+        width: 140,
+        height: 140,
+      },
+      {
+        size: "small",
+        url: faker.image.imageUrl(60, 60),
+        width: 60,
+        height: 60,
+      },
+      {
+        size: "tiny",
+        url: faker.image.imageUrl(30, 30),
+        width: 30,
+        height: 30,
+      }
+    ),
+  latitude: () => parseFloat(faker.address.latitude()),
+  longitude: () => parseFloat(faker.address.longitude()),
+  notes: () => randomObject(0, 5),
+  offset: () =>
+    faker.random.arrayElement(["+", "-"]) +
+    faker.random.number(11) +
+    ":" +
+    faker.random.arrayElement(["00", "30"]),
+  param: () => faker.name.title(),
+  seconds: () => faker.random.number(86400 - 30) + 30,
+  startEndSeconds: () => {
+    const min = faker.random.number(86400 - 30) + 30;
+    const max = faker.random.number(86400 - min) + min;
+    return [min, max];
+  },
+  startEndArray: (arr: any[]) => {
+    const min = faker.random.number(arr.length - 1);
+    const inc = faker.random.number(arr.length - 1 - min);
+    return [arr[min], arr[min + inc]];
+  },
+  timestamp: () => faker.date.past().toISOString(),
+  uuid: () => faker.random.uuid(),
+  timezone,
+  hexaDecimal,
+  randomArray,
+  randomObject,
+  ...faker,
+};
+
+/**
+ * Generate timezone data
+ */
+function timezone(): TimezoneInformation {
+  return faker.random.arrayElement<TimezoneInformation>([
+    {
+      identifierAlt: null,
+      identifier: "America/Costa_Rica",
+      friendlyIdentifier: "America - Costa Rica",
+      utcOffset: -21600,
+      utcTotalOffset: -21600,
+    },
+    {
+      identifierAlt: "Brisbane",
+      identifier: "Australia/Brisbane",
+      friendlyIdentifier: "Australia - Brisbane",
+      utcOffset: 36000,
+      utcTotalOffset: 36000,
+    },
+    {
+      identifierAlt: null,
+      identifier: "Asia/Makassar",
+      friendlyIdentifier: "Asia - Makassar",
+      utcOffset: 28800,
+      utcTotalOffset: 28800,
+    },
+  ]);
+}
+
+/**
+ * Generate a series of hexadecimal digits
+ * @param count Size of hash
+ */
+function hexaDecimal(count: number = 1): string {
+  let wholeString = "";
+  for (let i = 0; i < count; i++) {
+    wholeString += faker.random.arrayElement([
+      "0",
+      "1",
+      "2",
+      "3",
+      "4",
+      "5",
+      "6",
+      "7",
+      "8",
+      "9",
+      "a",
+      "b",
+      "c",
+      "d",
+      "e",
+      "f",
+      "A",
+      "B",
+      "C",
+      "D",
+      "E",
+      "F",
```

Comment:
no reason to have mixed case here. we usually default to lower case

Reply:
Twas just a copy paste from the faker.js implementation, if they ever make a new release it'll change back to this. I'll make the change anyways.